coil-sample/build.gradle.kts

```diff
@@ -10,15 +9,12 @@ plugins {
     id("kotlin-android")
 }
 
-android {
-    compileSdkVersion(project.compileSdk)
+setupAppModule {
     defaultConfig {
         applicationId = "coil.sample"
         minSdkVersion(16)
-        targetSdkVersion(project.targetSdk)
         versionCode = project.versionCode
         versionName = project.versionName
-        multiDexEnabled = true
```

Comment:
This is required.

Reply:
Ok, it was integrated into `setupAppModule` function